digital_image_processing/filters/laplacian_filter.py

```diff
@@ -15,8 +15,8 @@
 from gaussian_filter import gaussian_filter
 import numpy as np
 
```

Comment:
Two blank lines before a function definition.  psf/black should do this automatically.
```suggestion


```